src/operators.jl

```diff
@@ -92,6 +92,74 @@ function MA.operate_to!(
     return output
 end
 
+"""
+    _constant_term_idx(p::Polynomial)
+
+Return the index of the constant term in `p` according to its monomial ordering.
```

Comment:
It's not the constant term, it is the lowest term since there could also be no constant term.